Add color blindness tool to article (#26356)
mdn/content: files/en-us/web/accessibility/understanding_colors_and_luminance/index.md

Protanopia is a color vision deficiency in which the eye has no red cones; sRGB can still be perceived via green cones, though darker than normal vision. Both protan (red deficient) and deutan (green deficient) deficiencies cause difficulty distinguishing _between_ red and green.

> Developer tools can provide the ability to simulate color vision differences right in the browser. For example, Firefox enables similuating pronatopia, deuteranopia, tritanopia, achromatopsia, and contrast loss right in the developer tools, accessible in the accessibility pannel.

Reviewer: According to [this source](https://firefox-source-docs.mozilla.org/devtools-user/accessibility_inspector/simulation/index.html), this occurs in the Accessibility Inspector tool.  
```suggestion
Developer tools can help simulate color vision differences right in your browser. For example, Firefox's Accessibility Inspector enables simulating protanopia, deuteranopia, tritanopia, achromatopsia, and contrast loss right in the accessibility panel.
```
